BipedRobot/ZMPBipedRobot/docs/2. Optimisation process/objectives_values.txt

```diff
@@ -0,0 +1,2184 @@
+-Inf,Inf,Inf
```

Comment:
Could you think of a way to avoid to save this file in the repo itself? Is this something you could generate?

Reply:
I removed the docs folder however, to run the optimisation process can be quite long (>1h) to just generate the txt file. 
Actually, the code can't run the example 3 without running the example 2 code.

Reply:
If everything is explained in the README, it's all good :-)